dashboard/app/models/lesson.rb

```diff
@@ -181,9 +181,9 @@ def render_property(property_name)
   # page, and the lesson plan pdf as a backup.
   def start_url
     if script_levels.first
-      return url_from_path(build_script_level_path(script_levels.first))
+      return url_from_path(build_script_level_path(script_levels.first), 'https:')
```

Comment:
oops, my last comment belonged here:
```suggestion
      return url_from_path(build_script_level_path(script_levels.first), CDO.default_scheme)
```